core/trino-main/src/main/java/io/trino/sql/planner/RelationPlanner.java

```diff
@@ -267,7 +267,11 @@ public RelationPlan addRowFilters(Table node, RelationPlan plan, Function<Expres
     public RelationPlan addRowFilters(Table node, RelationPlan plan, Function<Expression, Expression> predicateTransformation, Function<Table, Scope> accessControlScope)
     {
         List<Expression> filters = analysis.getRowFilters(node);
+        return addRowFilters(filters, node, plan, predicateTransformation, accessControlScope);
+    }
 
+    public RelationPlan addRowFilters(List<Expression> filters, Table node, RelationPlan plan, Function<Expression, Expression> predicateTransformation, Function<Table, Scope> accessControlScope)
```

Comment:
for inserts/update/merge, the name `addRowConstraints`  would be better.
Row filters become "row constraints", that's fine.
(the opposite, calling "check constraints" a "row filter" doesn't sound right)